packages/web-components/fast-foundation/src/menu/menu.spec.ts

```diff
@@ -31,6 +31,7 @@ describe("Menu", () => {
     it("should set tabindex of the first focusable menu item to 0", async () => {
```

Comment:
Should there *also* be a test that a disabled item is focusable?

Reply:
One for each, sure.